util/runtime.go

```diff
@@ -0,0 +1,21 @@
+package util
+
+import (
+	"os"
+	"strings"
+)
+
+// Detect detects which Lambda runtime we're running in by checking the execution
+// environment variable. Should return a value such as "nodejs12.x" or "python3.6", or
+// "unknown" if runtime is not known.
+//
+// see: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime
+func DetectRuntime() string {
```

Comment:
Have you tested this? Docs [here](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-extensions-api.html) indicate that external extensions don't have access to runtime env vars

Reply:
It does look like extension access to `AWS_EXECUTION_ENV`, among other env vars, is specifically disallowed...but not all env vars are restricted.

Reply:
Yeah, @kolanos I will push up my code so that you can take a look. We've been duplicating efforts here

Reply:
@katiewest820 It looks like LAMBDA-990 is not possible without LAMBDA-989. I revised this pull request to do directory checking instead. When I noticed LAMBDA-990 was implementing the check I stopped any further work on a runtime-based check. What i have there doesn't check for the existence of the handler.